nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -431,6 +435,16 @@ in
         ";
       };
 
+      enableReload = mkOption {
+        default = false;
```

Comment:
I thought everybody agreed to make this true by default?

Reply:
:thinking: Maybe I'm just too cautious to change default behavior. Would you like to do that? :grin: